Combine tutorial and presentation pages into resources, add citation to references (#96)
ESMValGroup/ESMValTool-website: resources.md

---
layout: page
title: Resources
---

# Presentations

> Past Presentations using or introducing ESMValTool are listed in the [Community Repository](https://github.com/ESMValGroup/Community?tab=readme-ov-file#presentations) on GitHub. Some of these include links to the pdfs. If you would like to use those slides for your own presentation and would therefore require the .pptx files, feel free to contact the presenter given in the list.

Reviewer: ```suggestion
A list of past ESMValTool presentations can be found at the [Community Repository](https://github.com/ESMValGroup/Community?tab=readme-ov-file#presentations) on GitHub. Please feel free to use these slides in your own presentation. In case you prefer .pptx files, please contact the presenter given in the list.
```
